packages/oui-tabs/src/tabs.controller.js

```diff
@@ -50,5 +47,9 @@ export default class {
                 .addClass("oui-tabs")
                 .removeAttr("aria-label")
         );
+
+        if (this.items.length > 0) {
+            this.setActiveTab(find(this.items, tabItem => tabItem.id === this.model) || this.items[0]);
```

Comment:
```suggestion
            this.setActiveTab(find(this.items, {id: this.model}) || this.items[0]);
```
as you are using lodash 😉